### PR TITLE
Create south hemisphere moon visualization

### PR DESCRIPTION
**Pull request by martinbra**
Fixes issue #1. If closed, we can create a issue for the still missing _hemisphere autodetection_.

I had to fix moons lines lenght for the unequal length lines ending in `\ "` in order to correctly read the moon upside down.

I tried to keep the most of the code unchanged, and kept that long `if/elif` to get `char = background_nlines[lin][col]`. Was it in the original code?
Personally, I would change that to a dictionary to remove the if/elif and simplify it to `char = backgrounds[nlines][lin][col] if nlines in backgrounds.keys() else '@'`
If you like the idea, let me know and I can implement it.

Follows all the moons (first original and then upside down) for comparison and finally a waxing moon to show the change in direction:
```
    ..--..
  .` oo  .`.
 .o~.    O ).
 .c`_..() ||.
  `..o....'/
    `'--''

    ,,--,.
  /,````o``.
 `|| ()``^.c`
 `( O    `~o`
  `.`  oo .`
    ``--``


             .----------.
         .--'   o    .   `--.
       .'@  @@@@@@ O   .   . `.
     .'@@  @@@@@@@@   @@@@   . `.
   .'    . @@@@@@@@  @@@@@@    . `.
  / @@ o    @@@@@@.   @@@@    O   @\
  |@@@@               @@@@@@     @@|
 / @@@@@   `.-.    . @@@@@@@@  .  @@\
 | @@@@   --`-'  .  o  @@@@@@@      |
 |@ @@                 @@@@@@ @@@   |
 \      @@    @   . ()  @@   @@@@@  /
  |   @      @@@         @@@  @@@  |
  \  .   @@  @\  .      .  @@    o /
   `.   @@@@  _\ /     .      o  .'
     `.  @@    ()---           .'
       `.     / |  .    o    .'
         `--./   .       .--'
             `----------'

             ,----------.
         ,--`       `   /`--.
       ,`    o    `  | /     `.
     ,`           ---()    @@  `.
   ,`  o      `     / \^  @@@@   `.
  / o    @@  `      `  \@  @@   `  \
  |  @@@  @@@         @@@      @   |
 /  @@@@@   @@  () `   @    @@      \
 |   @@@ @@@@@@                 @@ @|
 |      @@@@@@@  o  `  ,-.--   @@@@ |
 \@@  `  @@@@@@@@ `    `-`.   @@@@@ /
  |@@     @@@@@@               @@@@|
  \@   O    @@@@   `@@@@@@    o @@ /
   `. `    @@@@@@  @@@@@@@@ `    ,`
     `. `   @@@@   @@@@@@@@  @@,`
       `. `   `   O @@@@@@  @,`
         `--.   `    o   ,--`
             `----------`

              .----------.
          .--'   o    .   `--.
       .-'@  @@@@@@ O   .   . `-.
     .' @@  @@@@@@@@   @@@@   .  `.
    /     . @@@@@@@@  @@@@@@     . \
   /@@  o    @@@@@@.   @@@@    O   @\
  /@@@@                @@@@@@     @@@\
 . @@@@@   `.-./    . @@@@@@@@  .  @@ .
 | @@@@   --`-'  .      @@@@@@@       |
 |@ @@        `      o  @@@@@@ @@@@   |
 |      @@        o      @@   @@@@@@  |
 ` .  @       @@     ()   @@@  @@@@   '
  \     @@   @@@@        . @@   .  o /
   \   @@@@  @@\  .           o     /
    \ . @@     _\ /    .      .-.  /
     `.    .    ()---        `-' .'
       `-.    ./ |  .   o     .-'
          `--./   .       .--'
              `----------'

              ,----------.
          ,--`       `   /`--.
       ,-`     o   `  | /`    `-.
     ,` ,-.        ---()    `    `.
    /  `-`      `    / \^     @@ ` \
   /     o           `  \@@  @@@@   \
  / o  `   @@ `        @@@@   @@     \
 ,   @@@@  @@@   ()     @@       @  ` .
 |  @@@@@@   @@      o        @@      |
 |   @@@@ @@@@@@  o      .        @@ @|
 |       @@@@@@@      `  ,-.--   @@@@ |
 ` @@  `  @@@@@@@@ `    /`-`.   @@@@@ `
  \@@@     @@@@@@                @@@@/
   \@   O    @@@@   `@@@@@@    o  @@/
    \ `     @@@@@@  @@@@@@@@ `     /
     `.  `   @@@@   @@@@@@@@  @@ ,`
       `-. `   `   O @@@@@@  @,-`
          `--.   `    o   ,--`
              `----------`

                .----------.
           .---'   O   . .  `---.
        .-'@ @@@@@@  .  @@@@@    `-.
      .'@@  @@@@@@@@@  @@@@@@@   .  `.
     /   o  @@@@@@@@@  @@@@@@@      . \
    /@  o   @@@@@@@@@.  @@@@@@@   O    \
   /@@@  .   @@@@@@o   @@@@@@@@@@     @@\
  /@@@@@            . @@@@@@@@@@@@@ o @@@\
 .@@@@@ O  `.-./ .     @@@@@@@@@@@@    @@ .
 | @@@@   --`-'      o    @@@@@@@@ @@@@   |
 |@ @@@       `   o     .  @@  . @@@@@@@  |
 |      @@  @        .-.    @@@  @@@@@@@  |
 `  . @       @@@    `-'  . @@@@  @@@@  o '
  \     @@   @@@@@ .         @@  .       /
   \   @@@@  @\@@    /  . O   .    o  . /
    \o  @@     \ \  /       .   .      /
     \    .    .\.-.___  .     .  .-. /
      `.         `-'             `-'.'
        `-.  o  / |    o   O  .  .-'
           `---.    .    .  .---'
                `----------'

                ,----------.
           ,---`  `    `    `---.
        ,-`  `  O   o    | /  o  `-.
      ,`,-.             ,-.         `.
     / `-`  `     `  ^^^`-`\`    `    \
    /      `   `       /  \ \     @@  o\
   / `  o    `   O `  /    @@\@  @@@@   \
  /       `  @@         ` @@@@@   @@     \
 , o  @@@@  @@@@ `  ,-.    @@@       @ `  .
 |  @@@@@@@  @@@    `-`        @  @@      |
 |  @@@@@@@ `  @@  `     o   .       @@@ @|
 |   @@@@ @@@@@@@@    o      ,-.--   @@@@ |
 ` @@    @@@@@@@@@@@@     ` /`-`.  O @@@@@`
  \@@@ o @@@@@@@@@@@@@ `            @@@@@/
   \@@     @@@@@@@@@@   o@@@@@@   `  @@@/
    \    O   @@@@@@@  `@@@@@@@@@   o  @/
     \ `      @@@@@@@  @@@@@@@@@  o   /
      `.  `   @@@@@@@  @@@@@@@@@  @@,`
        `-.    @@@@@  `  @@@@@@ @,-`
           `---.  ` `   O   ,---`
                `----------`

                .------------.
            .--'   o     . .  `--.
         .-'    .    O   .      . `-.
       .'@    @@@@@@@   .  @@@@@     `.
     .'@@@  @@@@@@@@@@@   @@@@@@@  .   `.
    /     o @@@@@@@@@@@   @@@@@@@      . \
   /@@  o   @@@@@@@@@@@.   @@@@@@@   O    \
  /@@@@   .   @@@@@@@o    @@@@@@@@@@    @@@\
  |@@@@@               . @@@@@@@@@@@@  @@@@|
 /@@@@@  O  `.-./  .      @@@@@@@@@@@   @@  \
 | @@@@    --`-'      o    . @@@@@@@ @@@@   |
 |@ @@@  @@  @ `   o  .-.     @@  . @@@@@@  |
 \             @@@    `-'  .   @@@  @@@@@@  /
  | . @  @@   @@@@@ .          @@@@  @@@ o |
  \     @@@@  @\@@    /  .  O   @@ .     . /
   \  o  @@     \ \  /          . . o     /
    \      .    .\.-.___   .  .  .  .-.  /
     `.           `-'              `-' .'
       `.    o   / |     o   O   .   .'
         `-.    /     .      .    .-'
            `--.        .     .--'
                `------------'

                ,------------.
            ,--`     `        `--.
         ,-`    `      `     /    `-.
       ,`   `   O   o     | /   o    `.
     ,` ,-.              ,-.           `.
    /  `-`  `  `  `   ^^^`-`\`    `      \
   /     o ` `          /  \ \     @@  o  \
  / `     ` @@   O  `  /    @@\@  @@@@     \
  | o @@@  @@@@          ` @@@@@   @@  @ ` |
 /  @@@@@@  @@@   `  ,-.    @@@             \
 |  @@@@@@ `  @@     `-`  o   . @  @@  @@@ @|
 |   @@@@ @@@@@@@ `    o      ,-.--    @@@@ |
 \  @@   @@@@@@@@@@@      `  /`-`.  O  @@@@@/
  |@@@@  @@@@@@@@@@@@ `               @@@@@|
  \@@@    @@@@@@@@@@    o@@@@@@@   `   @@@@/
   \    O   @@@@@@@   `@@@@@@@@@@@   o  @@/
    \ `      @@@@@@@   @@@@@@@@@@@ o     /
     `.   `  @@@@@@@   @@@@@@@@@@@  @@@,`
       `.     @@@@@  `   @@@@@@@    @,`
         `-. `      `   O    `    ,-`
            `--.  ` `     o   ,--`
                `------------`

                 .------------.
             .--'  o     . .   `--.
          .-'   .    O   .       . `-.
       .-'@   @@@@@@@   .  @@@@@      `-.
      /@@@  @@@@@@@@@@@   @@@@@@@   .    \
    ./    o @@@@@@@@@@@   @@@@@@@       . \.
   /@@  o   @@@@@@@@@@@.   @@@@@@@   O      \
  /@@@@   .   @@@@@@@o    @@@@@@@@@@     @@@ \
  |@@@@@               . @@@@@@@@@@@@@ o @@@@|
 /@@@@@  O  `.-./  .      @@@@@@@@@@@@    @@  \
 | @@@@    --`-'       o     @@@@@@@@ @@@@    |
 |@ @@@        `    o      .  @@   . @@@@@@@  |
 |       @@  @         .-.     @@@   @@@@@@@  |
 \  . @        @@@     `-'   . @@@@   @@@@  o /
  |      @@   @@@@@ .           @@   .       |
  \     @@@@  @\@@    /  .  O    .     o   . /
   \  o  @@     \ \  /         .    .       /
    `\     .    .\.-.___   .      .   .-. /'
      \           `-'                `-' /
       `-.   o   / |     o    O   .   .-'
          `-.   /     .       .    .-'
             `--.       .      .--'
                 `------------'

                 ,------------.
             ,--`      `       `--.
          ,-`    `       `     /   `-.
       ,-`   `   O    o     | /   o   `-.
      / ,-.                ,-.           \
    ,/ `-`   `      `   ^^^`-`\`    `     \.
   /       `    `         /  \ \     @@  o  \
  / `   o     `    O  `  /    @@\@  @@@@     \
  |       `   @@           ` @@@@@   @@      |
 / o  @@@@   @@@@ `   ,-.     @@@        @ `  \
 |  @@@@@@@   @@@     `-`         @  @@       |
 |  @@@@@@@ `   @@  `      o    .        @@@ @|
 |    @@@@ @@@@@@@@     o       ,-.--    @@@@ |
 \  @@    @@@@@@@@@@@@      `  /`-`.  O  @@@@@/
  |@@@@ o @@@@@@@@@@@@@ `               @@@@@|
  \ @@@     @@@@@@@@@@    o@@@@@@@   `   @@@@/
   \      O   @@@@@@@   `@@@@@@@@@@@   o  @@/
    `\ `       @@@@@@@   @@@@@@@@@@@ o    /`
      \    `   @@@@@@@   @@@@@@@@@@@  @@@/
       `-.      @@@@@  `   @@@@@@@   @,-`
          `-. `       `   O    `   ,-`
             `--.   ` `     o  ,--`
                 `------------`

                  .------------.
             .---' o     .  .   `---.
          .-'   .    O    .       .  `-.
        .'@   @@@@@@@   .   @@@@@       `.
      .'@@  @@@@@@@@@@@    @@@@@@@   .    `.
     /    o @@@@@@@@@@@    @@@@@@@       .  \
    /@  o   @@@@@@@@@@@.    @@@@@@@   O      \
   /@@@   .   @@@@@@@o     @@@@@@@@@@     @@@ \
  /@@@@@               .  @@@@@@@@@@@@@ o @@@@ \
  |@@@@  O  `.-./  .       @@@@@@@@@@@@    @@  |
 / @@@@    --`-'       o      @@@@@@@@ @@@@     \
 |@ @@@     @  `           .   @@     @@@@@@@   |
 |      @           o          @      @@@@@@@   |
 \       @@            .-.      @@@    @@@@  o  /
  | . @        @@@     `-'    . @@@@           |
  \      @@   @@@@@ .            @@   .        /
   \    @@@@  @\@@    /  .   O    .     o   . /
    \ o  @@     \ \  /          .    .       /
     \     .    .\.-.___    .      .   .-.  /
      `.          `-'                 `-' .'
        `.   o   / |      o    O   .    .'
          `-.   /      .       .     .-'
             `---.       .      .---'
                  `------------'

                  ,------------.
             ,---`      `       `---.
          ,-`     `       `      /   `-.
        ,`    `   O    o      | /   o   `.
      ,` ,-.                 ,-.          `.
     /  `-`   `      `    ^^^`-`\`    `     \
    /       `    `          /  \ \     @@  o \
   / `   o     `    O   `  /    @@\@  @@@@    \
  /        `   @@            ` @@@@@   @@      \
  |           @@@@ `    ,-.     @@@        @ ` |
 /  o  @@@@    @@@      `-`            @@       \
 |   @@@@@@@      @          o           @      |
 |   @@@@@@@     @@   `           .  @     @@@ @|
 \     @@@@ @@@@@@@@      o       ,-.--    @@@@ /
  |  @@    @@@@@@@@@@@@       `  /`-`.  O  @@@@|
  \ @@@@ o @@@@@@@@@@@@@  `               @@@@@/
   \ @@@     @@@@@@@@@@     o@@@@@@@   `   @@@/
    \      O   @@@@@@@    `@@@@@@@@@@@   o  @/
     \  `       @@@@@@@    @@@@@@@@@@@ o    /
      `.    `   @@@@@@@    @@@@@@@@@@@  @@,`
        `.       @@@@@   `   @@@@@@@   @,`
          `-.  `       `    O    `   ,-`
             `---.   `  `     o ,---`
                  `------------`

                      .--------------.
                 .---'  o        .    `---.
              .-'    .    O  .         .   `-.
           .-'     @@@@@@       .             `-.
         .'@@   @@@@@@@@@@@       @@@@@@@   .    `.
       .'@@@  @@@@@@@@@@@@@@     @@@@@@@@@         `.
      /@@@  o @@@@@@@@@@@@@@     @@@@@@@@@     O     \
     /        @@@@@@@@@@@@@@  @   @@@@@@@@@ @@     .  \
    /@  o      @@@@@@@@@@@   .  @@  @@@@@@@@@@@     @@ \
   /@@@      .   @@@@@@ o       @  @@@@@@@@@@@@@ o @@@@ \
  /@@@@@                  @ .      @@@@@@@@@@@@@@  @@@@@ \
  |@@@@@    O    `.-./  .        .  @@@@@@@@@@@@@   @@@  |
 / @@@@@        --`-'       o        @@@@@@@@@@@ @@@    . \
 |@ @@@@ .  @  @    `    @            @@      . @@@@@@    |
 |   @@                         o    @@   .     @@@@@@    |
 |  .     @   @ @       o              @@   o   @@@@@@.   |
 \     @    @       @       .-.       @@@@       @@@      /
  |  @    @  @              `-'     . @@@@     .    .    |
  \ .  o       @  @@@@  .              @@  .           . /
   \      @@@    @@@@@@       .                   o     /
    \    @@@@@   @@\@@    /        O          .        /
     \ o  @@@       \ \  /  __        .   .     .--.  /
      \      .     . \.-.---                   `--'  /
       `.             `-'      .                   .'
         `.    o     / | `           O     .     .'
           `-.      /  |        o             .-'
              `-.          .         .     .-'
                 `---.        .       .---'
                      `--------------'

                      ,--------------.
                 ,---`       `        `---.
              ,-`     `         `          `-.
           ,-`             o        |  /      `-.
         ,`     `     O           . | /     o    `.
       ,`                   `      ,-.             `.
      /  ,--.                   ---`-`\ `     `      \
     /  `--`     `   `        ^^  /  \ \       @@@  o \
    /        `          O        /    @@\@@   @@@@@    \
   /     o                   `       @@@@@@    @@@      \
  / `           `  @@              `  @@@@  @       o  ` \
  |    `    `     @@@@ `     ,-.              @  @    @  |
 /      @@@       @@@@       `-`       @       @    @     \
 |   `@@@@@@   o   @@              o       @ @   @     `  |
 |    @@@@@@     `   @@    o                         @@   |
 |    @@@@@@ `      @@            @    .    @  @  ` @@@@ @|
 \ `    @@@ @@@@@@@@@@@        o       ,-.--        @@@@@ /
  |  @@@   @@@@@@@@@@@@@  `        `  /`-`.    O    @@@@@|
  \ @@@@@  @@@@@@@@@@@@@@      ` @                  @@@@@/
   \ @@@@ o @@@@@@@@@@@@@  @       o @@@@@@   `      @@@/
    \ @@     @@@@@@@@@@@  @@  `   @@@@@@@@@@@      o  @/
     \  `     @@ @@@@@@@@@   @  @@@@@@@@@@@@@@        /
      \     O     @@@@@@@@@     @@@@@@@@@@@@@@ o  @@@/
       `.         @@@@@@@@@     @@@@@@@@@@@@@@  @@@,`
         `.    `   @@@@@@@       @@@@@@@@@@@   @@,`
           `-.             `       @@@@@@     ,-`
              `-.   `         `  O    `    ,-`
                 `---.    `        o  ,---`
                      `--------------`

                         .--------------.
                   .----'  o        .    `----.
                .-'     .    O  .          .   `-.
             .-'      @@@@@@       .              `-.
           .'@     @@@@@@@@@@@       @@@@@@@@    .   `.
         .'@@    @@@@@@@@@@@@@@     @@@@@@@@@@         `.
       .'@@@ o   @@@@@@@@@@@@@@     @@@@@@@@@@      o    `.
      /@@@       @@@@@@@@@@@@@@  @   @@@@@@@@@@  @@     .  \
     /            @@@@@@@@@@@   .  @@   @@@@@@@@@@@@     @@ \
    /@  o     .     @@@@@@ o       @   @@@@@@@@@@@@@@ o @@@@ \
   /@@@                        .       @@@@@@@@@@@@@@@  @@@@@ \
  /@@@@@                     @      .   @@@@@@@@@@@@@@   @@@   \
  |@@@@@     o      `.-./  .             @@@@@@@@@@@@ @@@    . |
 / @@@@@           __`-'       o          @@       . @@@@@@     \
 |@ @@@@ .        @    `    @            @@    .     @@@@@@     |
 |   @@       @                    o       @@@   o   @@@@@@.    |
 |          @                             @@@@@       @@@       |
 |  . .  @      @  @       o              @@@@@     .    .      |
 \            @                .-.      .  @@@  .           .   /
  |    @   @   @      @        `-'                     .       /
  \   .      @   @                   .            o            /
   \     o          @@@@   .                .                 /
    \       @@@    @@@@@@        .                    o      /
     \     @@@@@   @@\@@    /         o           .         /
      \  o  @@@       \ \  /  ___         .   .     .--.   /
       `.      .       \.-.---                     `--'  .'
         `.             `-'       .                    .'
           `.    o     / |              O      .     .'
             `-.      /  |         o              .-'
                `-.           .         .      .-'
                   `----.        .       .----'
                         `--------------'

                         ,--------------.
                   ,----`       `        `----.
                ,-`      `         `           `-.
             ,-`              o         |  /      `-.
           ,`     `      O              | /     o    `.
         ,`                    `       ,-.             `.
       ,`  ,--.                     ---`-`\       `      `.
      /   `--`     `   `         ^^^  /  \ \       @@@  o  \
     /         `           o         /    @@\@@   @@@@@     \
    /      o                    `        @@@@@@    @@@       \
   /                 `                `   @@@@          o     \
  /            o            `                   @   @      `   \
  /       `                     ,-.        @      @   @   @    |
 /   `           `  @@@  `      `-`                @            \
 |      `    `     @@@@@              o       @  @      @  ` `  |
 |       @@@       @@@@@                             @          |
 |    `@@@@@@   o   @@@       o                    @       @@   |
 |     @@@@@@     `    @@            @    .    @        ` @@@@ @|
 \     @@@@@@ `       @@          o       ,-.^^           @@@@@ /
  | `    @@@ @@@@@@@@@@@@             `  /`-`.      o     @@@@@|
  \   @@@   @@@@@@@@@@@@@@   `      @                     @@@@@/
   \ @@@@@  @@@@@@@@@@@@@@@       `                        @@@/
    \ @@@@ o @@@@@@@@@@@@@@   @       o @@@@@@     `     o  @/
     \ @@     @@@@@@@@@@@@   @@  `   @@@@@@@@@@@            /
      \  `     @@  @@@@@@@@@@   @  @@@@@@@@@@@@@@       @@@/
       `.    o      @@@@@@@@@@     @@@@@@@@@@@@@@   o @@@,`
         `.         @@@@@@@@@@     @@@@@@@@@@@@@@    @@,`
           `.   `    @@@@@@@@       @@@@@@@@@@@     @,`
             `-.              `       @@@@@@      ,-`
                `-.   `          `  O    `     ,-`
                   `----.    `        o  ,----`
                         `--------------`

             .---
         .--'  
       .'@  @
     .'@@  @
   .'    . @
  / @@ o   
  |@@@@    
 / @@@@@   
 | @@@@   
 |@ @@    
 \      @@ 
  |   @    
  \  .   @@
   `.   @@@@
     `.  @@ 
       `.    
         `--./ 
             `---

                     ---.
                        /`--.
                             `.
                           @@  `.
                          @@@@   `.
                           @@   `  \
                               @   |
                            @@      \
                                @@ @|
                               @@@@ |
                              @@@@@ /
                               @@@@|
                              o @@ /
                          @ `    ,`
                          @  @@,`
                         @  @,`
                         ,--`
                     ---`

```

### CODE REVIEW

**Comment by martinbra** [comment]
Oh, I forgot to update the version number.

File: src/__init__.py [hunk 1]
```diff
@@ -216,4 +261,11 @@ def main():
...
 
-    print(putmoon(dateobj, numlines, '@', notext, lang))
+    try:
+        hemisphere = str(args['hemisphere'])
+    except Exception as err:  # pylint: disable=broad-except
+        print(err)
+
+    print(putmoon(dateobj, numlines, '@', notext, lang, hemisphere))
+
+main()
```
Comment (martinbra) [inline src/__init__.py:271]:
```suggestion

```
I should remove the main call here. I added before, and its giving the output twice.

File: src/__init__.py [hunk 2]
```diff
@@ -121,26 +129,55 @@ def fputs(string):
...
+                elif numlines == 32:
+                    char = background32[lin][col]
+                else:
+                    char = '@'
             else:
-                char = '@'
+                # south - read moons from bottom-right to upper-left
+                # equivalent to rotate 180 degress or turn upside-down
+                if numlines == 6:
+                    char = background6[-1-lin][-col-1]
```
Comment (martinbra) [inline src/__init__.py:158]: this line fixes the 6 line moon beeing incorrectly displayed, but I want to further improve or review why it was needed to shift the column.

File: src/lib/rotate.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    "'":','
+    }
+
+def rotate(char):
+    """ Rotate char if it has a "symmetric" char. 
+    """
+    if char in swapTable.keys():
+        return swapTable[char]
+    else:
+        return char
```
Comment (chubin) [inline src/lib/rotate.py]: You can simply write `return swapTable.get(char, char)` instead, it is the same.
Also you can use `translate` method of `string` here (or even where the `rotate` function is called).

https://www.programiz.com/python-programming/methods/string/translate

File: src/__init__.py [hunk 4]
```diff
@@ -121,26 +129,55 @@ def fputs(string):
...
+                elif numlines == 23:
+                    char = background23[-1-lin][-col]
+                elif numlines == 24:
+                    char = background24[-1-lin][-col]
+                elif numlines == 29:
+                    char = background29[-1-lin][-col]
+                elif numlines == 32:
+                    char = background32[-1-lin][-col]
+                else:
+                    char = '@'
```
Comment (chubin) [inline src/__init__.py:176]: Actually, my original idea was to stay to the 1973 `cphoon` of Jeff as close as possible,
but if we change the code anyway we can write this code much more concise (in Python).
Or maybe you are right, and we should leave the spirit of the code as is

**Comment by martinbra** [comment]
@chubin Thanks for your review and valuable tips.  I also think that we should try to keep the original spirit alive.
I reviewed the points we mentioned and would appreciate if you could have another look. 
I also wonder what your thoughts are on the `--hemisphere` short switch argument `-s` (from _sphere_, since `-h` is taken for the help command). Do you think it's ok? Or should we use `-e` or another letter?

**Comment by chubin** [comment]
It looks good to me. Great work, Martin!
I would merge it and then let's make the next step: enable it in wttr.in

**Pull request merged**